MessengerOnFileForLinux/DisplayChat/src/ConsoleWindow.cpp

```diff
@@ -26,34 +26,62 @@ void ConsoleWindow::displayRegistrationWindow()
     move(0,0);
     printw("Hello ");
     printw(getenv("USER"));
-    printw(". You are using messenger first time. You need create account.");
+    printw(". You are using messenger first time. You need create account. Enter your password twice.");
```

Comment:
wszystkie komunikaty zebrać w jeden plik w Common i po prostu wywoływać np printw(msgInfoMsg::pleaseRegister)

Reply:
Tak, właśnie miałem podobny pomysł w związku z tymi problematycznymi testami z SignIn i RegisterIn. Ale to zajmę się tym w obrębie tamtego taska

Reply:
ok